docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>I'd probably go ahead and add a header here to summarize what this part of the tutorial is about (maybe "Accessing and Filtering the Items" or something just to kind of separate it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>